src/App.js

```diff
@@ -16,11 +21,11 @@ function App() {
   const responseType = "token";
   const scope = "user-top-read";
 
+  // Token needed for Oauth
   const [token, setToken] = useState("");
-  const [searchKey, setSearchKey] = useState("");
-  const [tracks, setTracks] = useState([]);
   const [playlist, setPlaylist] = useState([]);
```

Comment:
We need to keep these hooks in this file - they're storing and setting some of the key variables we need to use e.g. playlist